android/src/main/java/io/ably/flutter/plugin/AblyLibrary.java

```diff
@@ -1,31 +1,37 @@
 package io.ably.flutter.plugin;
 
+import android.content.Context;
 import android.util.LongSparseArray;
 
 import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.rest.AblyBase;
 import io.ably.lib.rest.AblyRest;
 import io.ably.lib.types.AblyException;
 import io.ably.lib.types.AsyncPaginatedResult;
 import io.ably.lib.types.ClientOptions;
+import io.ably.lib.types.ErrorInfo;
 
-class AblyLibrary {
+public class AblyLibrary {
```

Comment:
I'm curious as to why package scope wasn't sufficient.

Reply:
When I added `public` to AblyLibrary, it temporarily tricked Android Studio to resolving the `AblyLibrary` symbol in other files. Annoyingly, this fix is not permanent, so there is no reason to keep this. Reverted in 2000557a1a21ed289e33820de5b93e601165e43f